java-core/google-cloud-core-http/src/test/java/com/google/cloud/http/HttpTransportOptionsTest.java

```diff
@@ -42,6 +77,144 @@ public class HttpTransportOptionsTest {
   private static final HttpTransportOptions DEFAULT_OPTIONS =
       HttpTransportOptions.newBuilder().build();
   private static final HttpTransportOptions OPTIONS_COPY = OPTIONS.toBuilder().build();
+  private static final String DEFAULT_PROJECT_ID = "testing";
+  private static final String CUSTOM_UNIVERSE_DOMAIN = "random.com";
+
+  private HeaderProvider defaultHeaderProvider;
+  // Credentials' getUniverseDomain() returns GDU
+  private Credentials defaultCredentials;
+  // Credentials' getUniverseDomain() returns `random.com`
+  private Credentials customCredentials;
+  private HttpRequest defaultHttpRequest;
+
+  /**
+   * The following interfaces and classes are from ServiceOptionsTest. Copied over here as
+   * ServiceOptions resides inside google-cloud-core test folder and is not accessible from
+   * google-cloud-core-http.
+   */
+  interface TestService extends Service<TestServiceOptions> {}
```

Comment:
nit: It might be subjective, usually I prefer to put these private test interfaces/classes to the bottom of the file so it's easier to focus on the actual test cases.